drivers/lsm6dsox/lsm6dsox.py

```diff
@@ -4,8 +4,8 @@
 
 The MIT License (MIT)
 
-Copyright (c) 2021 Damien P. George
-Copyright (c) 2021 Ibrahim Abdelkader <iabdalkader@openmv.io>
+Copyright (c) 2022 Damien P. George
+Copyright (c) 2022 Ibrahim Abdelkader <iabdalkader@openmv.io>
```

Comment:
This file was created in 2021, so the copyright should stay at that for me.  For yourself, you can change it to `2021-2022`.